fixtures/contacts.yaml

```diff
@@ -3,9 +3,9 @@ contacts:
         name: 311
         email: help@austintexas.gov
         phone:
-            default: 311
-            out-of-area: 512-974-2000
-            tdd: 512-972-9848
+            default: 512-974-2000
+            n11: 311
```

Comment:
this might be a dumb, but is n11 specific to 311? will this key be changing for different contacts/phone numbers?

Reply:
`n11` is just the term for 311, 911, etc. It sounded like we weren't going to display `311` so i removed it from default.